front/app/utils/permissions/rules/campaignPermissions.ts

```diff
@@ -11,5 +11,5 @@ definePermissionRule(
 );
 
 definePermissionRule('manualCampaign', 'manage', (_campaign: string, user) => {
-  return isAdmin(user) || isProjectModerator(user);
```

Comment:
I don't know if this has changed tho with recent project moderator work?

Reply:
As far as I can see, `/admin/messaging` is still only accessible to admins (it's not part of `MODERATOR_ROUTES`). Given this permission rule is only used in the component that's rendered on the index there, I think it's not doing anything at the moment. I.e., if you can visit the messaging route, you're already an admin. And the checks we're doing with this rule are pointless. :)

Reply:
In any case, I'll not touch this in this PR to not further complicate it. Will do this on a separate branch/ticket (and assign other people to spread the burden). :)